adapters/http-vertx-base/src/main/java/org/eclipse/hono/adapter/http/HonoHttpAuthenticationHandler.java

```diff
@@ -27,21 +27,21 @@
 import io.vertx.core.Future;
 import io.vertx.core.Handler;
 import io.vertx.core.json.JsonObject;
-import io.vertx.ext.auth.AuthProvider;
+import io.vertx.ext.auth.authentication.AuthenticationProvider;
 import io.vertx.ext.web.RoutingContext;
 
 /**
- * Interface for Hono {@link io.vertx.ext.web.handler.AuthHandler} implementations.
+ * Interface for Hono {@link io.vertx.ext.web.handler.AuthenticationHandler} implementations.
  *
  */
-public interface HonoHttpAuthHandler {
+public interface HonoHttpAuthenticationHandler {
```

Comment:
This interface seems obsolete now - it isn't used and therefore could be removed.

Reply:
And then `org.eclipse.hono.adapter.http.HttpAuthProviderAdapter` can be removed as well.